src/client/utils/validations.js

```diff
@@ -239,8 +239,15 @@ function isValidVeteranAddress(data) {
 }
 
 function isValidContactInformationSection(data) {
+  let emailConfirmationValid = true;
+
+  if (isNotBlank(data.email.value) && isBlank(data.emailConfirmation.value)) {
+    emailConfirmationValid = false;
+  }
```

Comment:
this looks like it only validates the presence of both fields (email and email confirmation), not that they are both equal. is that validation somewhere else?